proto/injective/feemarket/v1/genesis.proto

```
@@ -0,0 +1,20 @@
+syntax = "proto3";
+package injective.feemarket.v1;
+
+import "injective/feemarket/v1/feemarket.proto";
```

buf (1.47.2)

import "injective/feemarket/v1/feemarket.proto": file does not exist

(COMPILE)